.github/workflows/ci.yml

```diff
@@ -51,6 +51,12 @@ jobs:
         with:
           go-version-file: 'go.mod'
 
+      - name: Generate embed file
+        run: go generate ./...
+
+      - name: Launch golangci-lint
+        uses: golangci/golangci-lint-action@v6
+
```

Comment:
Can you move this into a new job instead?
This way, `golang-ci-lint` failing doesn't prevent the `lint` job from running.